invenio_rdm_records/models.py

```diff
@@ -34,6 +34,8 @@ class BibliographicRecordDraft(DraftBase):
 class BibliographicRecord(Record):
     """Bibliographic Record API."""
 
+    index = IndexField('records-record-v1.0.0', search_alias='records')
```

Comment:
minor: this should come from `Record`.

Reply:
Are we indexing all different kinds of records in the same index? Authors and BibliographicRecords?

Reply:
You are right, good point